docs/structured-streaming-programming-guide.md

```diff
@@ -3047,12 +3047,6 @@ To run a supported query in continuous processing mode, all you need to do is sp
 {% highlight scala %}
 import org.apache.spark.sql.streaming.Trigger
 
-spark
-  .readStream
-  .format("rate")
-  .option("rowsPerSecond", "10")
-  .option("")
-
```

Comment:
This seems to be incomplete leftover. We had better remove this because the other language tabs don't have this. After removing this, we will have `Kafka` examples consistently across all languages.